docs/guides/getting-started/setup/vite.mdx

```diff
@@ -56,13 +56,13 @@ pnpm create vite
   </TabItem>
 </Tabs>
 
-1. **Project name**  
+1. **Project name**
```

Comment:
```suggestion
1. _Project name_  
```

Reply:
ah shit, i edited the vite guide before i disabled vscode's traling whitespace auto format stuff...

Reply:
I have an mdx plugin in vs code that fixes some of those things, just basically treats .mdx as .md and stops weird things from happening

Reply:
yeah i set that up (without a plugin) now, because i needed it for the sveltekit guide but forgot that i changed the vite guide earlier :/